addons/product/models/product_category.py

```diff
@@ -7,6 +7,7 @@
 
 class ProductCategory(models.Model):
     _name = "product.category"
+    _inherit = ['mail.thread', 'mail.activity.mixin']
```

Comment:
Do we also need activities on the product categories ?
AFAIK, there is no mention of that in the task :eyes: